### PR TITLE
Fix stacking of blast doors and shutters from crafting

### PR DESCRIPTION
**Pull request by Ghilker**
## About The Pull Request
Fix stacking exploit of blast doors and shutters, only one per turf now

## Why It's Good For The Game
Exploit fix

## Changelog
:cl:
fix: fixed blast doors and shutters stacking exploit
/:cl:

### CODE REVIEW

**Comment by ATH1909** [comment]
aw

**Comment by ATH1909** [comment]
recreating the wine press from hitman 3 or that hydraulic press that's featured in those youtube videos sounds fun :(

**Comment by Ghilker** [comment]
> recreating the wine press from hitman 3 or that hydraulic press that's featured in those youtube videos sounds fun :(

sorry but we already removed many other stacking issues, this one had to go too

**Comment by TheBonded** [comment]
## I think this should stay, and here's why:

## 1. Blast doors take a second to close
When you push a button to close blast doors or shutters, there is an animation of them closing. This gives anyone inside them ample time to escape. If they can't move, they are already trapped to killing them with blast doors is no different than any other method.

## 2. Blast doors take a lot to make
Taking 15 plasteel and 30-45 seconds to make, they are time and material consuming, making other forms of murder far more efficient when compared to blast doors. It takes between 8 and 13 minutes as well as 255 plasteel sheets and 255 cable coil to make enough blast doors to kill someone in one close for one tile. If you want to cover more than that tile, you'd better have another 255 plasteel and 255 cable coil to make another 17 blast doors.

## 3. There are more efficient ways to kill someone
Seriously, with all the chemical mixes, xenobio combinations, or even just a stun prod and a toolbox can kill someone way easier than trying to get them onto that one spot in maint you have set up your stacked blast doors.

## 4. There are more efficient ways of making hard to destroy walls
Rwalls (reinforced walls) can have grilles and glass on them. An r-wall with a grille and a plastitanium window on top is just as hard to destroy than several stacked blast doors, and uses up far less materials. An rwall takes 2 plasteel, and a blast door takes 15. Just two stacked blast doors takes enough plasteel to make 15 reinforced walls. Both are just as easily destroyed.

## 5. Blast doors don't move
What.
How.
That's right, you can't move your stacked blast doors over people. If you want to crush someone you have to move them onto your blast doors yourself.

## 6. They're fun
Stacked blast doors are fun! Recreate that wine press from hitman 3, make a monkey crushing factory, if you put in the time you'll get your fair share of fun.

## In conclusion
Stacked blast doors (and shutters) are a good thing. I believe they are balanced, because of the time and materials that need to be put into using them, and because of their niche scenarios that they are actually used effectively

**Comment by TiviPlus** [comment]
B+
See me after class

**Comment by ATH1909** [comment]
I mean, he's not wrong.

**Comment by LemonInTheDark** [comment]
> Lots Of Shit

Short answer: Lol No

Long answer: Just cause something is neat/useful/balanced in comparison to other options that doesn't mean we should keep it. While there are some bugs that lead to unique behavior that's worth keeping, it's kinda a matter of personal opinion, and in mine, this is way too far outside "normal"

**Comment by Mothblocks** [comment]
a short story to rival "for sale: baby shoes, never worn"

image

**Comment by Mothblocks** [comment]
also im gonna be honest i didnt read any of that message i just woke up sorry

**Comment by TheBonded** [comment]
> also im gonna be honest i didnt read any of that message i just woke up sorry

@Mothblocks  I bet if you read my 7 paragraph response you would consider reverting this

**Comment by Mothblocks** [comment]
thats a big if

**Comment by TheBonded** [comment]
> thats a big if

It's literally like a 30 second to 2 minute read

**Comment by Mothblocks** [comment]
yeah
